src/config-utils.test.ts

```diff
@@ -913,6 +925,89 @@ test("Queries in workflow file can be added to the set of queries without overri
   });
 });
 
+test("Queries can be specified in configuration, same as file", async (t) => {
+  return await util.withTmpDir(async (tmpDir) => {
+    const inputFileContents = `
+      name: my config
+      queries:
+        - uses: ./foo
+      packs:
+        javascript:
+          - a/b@1.2.3
+        python:
+          - c/d@1.2.3
+    `;
+
+    fs.mkdirSync(path.join(tmpDir, "foo"));
+
+    const resolveQueriesArgs: Array<{
+      queries: string[];
+      extraSearchPath: string | undefined;
+    }> = [];
+    const codeQL = setCodeQL({
+      async resolveQueries(
+        queries: string[],
+        extraSearchPath: string | undefined
+      ) {
+        resolveQueriesArgs.push({ queries, extraSearchPath });
+        return queriesToResolvedQueryForm(queries);
+      },
+      async packDownload(): Promise<PackDownloadOutput> {
+        return { packs: [] };
+      },
+    });
+
+    // Only JS, python packs will be ignored
+    const languages = "javascript";
+
+    const config = await configUtils.initConfig(
+      languages,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      inputFileContents,
+      false,
+      false,
+      "",
+      "",
+      { owner: "github", repo: "example " },
```

Comment:
Copied and pasted, but there shouldn't be a space here.  Please could you correct this throughout the file?

```suggestion
      { owner: "github", repo: "example" },
```